lib/tasks/analyse/links.rb

```diff
@@ -81,7 +81,7 @@ def write_taxon_worksheet(results, spreadsheet)
     results_by_taxon_base_path.each_pair do |taxon_base_path, base_path_results|
       navigation_page_type = base_path_results[0][:navigation_page_type]
 
-      if navigation_page_type == 'accordion'
+      if navigation_page_type == 'accordion' || 'grid'
```

Comment:
(At this point it might be easier to say `unless navigation_page_type == 'leaf'`?)